querybean-generator/src/main/java/io/ebean/querybean/generator/ProcessingContext.java

```diff
@@ -16,118 +33,68 @@
 import javax.lang.model.type.TypeKind;
 import javax.lang.model.type.TypeMirror;
 import javax.lang.model.util.ElementFilter;
-import javax.lang.model.util.Elements;
-import javax.lang.model.util.Types;
-import javax.tools.Diagnostic;
 import javax.tools.FileObject;
-import javax.tools.JavaFileObject;
 import javax.tools.StandardLocation;
-import java.io.FileNotFoundException;
-import java.io.IOException;
-import java.io.LineNumberReader;
-import java.io.Reader;
-import java.nio.file.NoSuchFileException;
-import java.util.ArrayList;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
-import java.util.TreeMap;
-import java.util.TreeSet;
 
-/**
- * Context for the source generation.
- */
+/** Context for the source generation. */
 class ProcessingContext implements Constants {
 
-  private final ProcessingEnvironment processingEnv;
-  private final Types typeUtils;
-  private final Filer filer;
-  private final Messager messager;
-  private final Elements elementUtils;
-
-  private final PropertyTypeMap propertyTypeMap = new PropertyTypeMap();
+  private static final ThreadLocal<Ctx> CTX = new ThreadLocal<>();
 
-  private final ReadModuleInfo readModuleInfo;
-
-  /**
-   * All entity packages regardless of DB (for META-INF/ebean-generated-info.mf).
-   */
-  private final Set<String> allEntityPackages = new TreeSet<>();
+  private static final class Ctx {
+    private final Set<String> services = new TreeSet<>();
 
-  private final Set<String> otherClasses = new TreeSet<>();
+    private final PropertyTypeMap propertyTypeMap = new PropertyTypeMap();
 
-  /**
-   * The DB name prefixed entities.
-   */
-  private final Set<String> prefixEntities = new TreeSet<>();
+    /** All entity packages regardless of DB (for META-INF/ebean-generated-info.mf). */
```

Comment:
If you put the formatting back to the original ... then this does not become a DIFF ... and that helps me review it. Adding these types of DIFFs does not help a big PR.

Reply:
all of these fields are going into a nested class class, so formatting has to change no?